docs/dispatcher/filters/magic_filters.rst

```diff
@@ -117,7 +116,7 @@ All operations can be combined via bitwise and/or operators - :code:`&`/:code:`|
 
     (F.from_user.id == 42) & (F.text == 'admin')
     F.text.startswith('a') | F.text.endswith('b')
-    (F.from_user.id @ {42, 777, 911}) & (F.text.startswith('!') | F.text.startswith('/')) & F.text.contains('ban')
+    (F.from_user.id.in_([42, 777, 911])) & (F.text.startswith('!') | F.text.startswith('/')) & F.text.contains('ban')
```

Comment:
Why `Set` was changed to `List`?
For this kind of usage sets are more efficient

Reply:
It's seems like a typing problem on method side